spec/spec_helper.rb

```diff
@@ -9,17 +9,18 @@
     mocks.verify_partial_doubles = true
   end
 
-  config.shared_context_metadata_behavior = :apply_to_host_groups
-
-  config.filter_run_when_matching :focus
+  config.filter_run :focus
+  config.run_all_when_everything_filtered = true
 
   config.disable_monkey_patching!
 
   config.warnings = true
 
-  config.default_formatter = 'doc' if config.files_to_run.one?
+  if config.files_to_run.one?
```

Comment:
[Rubocop](https://github.com/rubocop-hq/ruby-style-guide#if-as-a-modifier) says: ```Favor modifier if usage when having a single-line body.```

Reply:
I was keeping parity with the `wisper-sidekiq` gem, but I don't think that project is enforcing style with rubocop. I will update this.

Reply:
I'll setup Travis with Rubocop + RSpec afterwards.